src/components/NonProfitApply/NonProfitApply.js

```diff
@@ -143,10 +141,12 @@ export default function NonProfitApply() {
 
     fetch("http://localhost:3001/submit-application", options)
       .then((response) => {
-        return response.json();
+        if (response.ok) {
+          router.push("http://localhost:3000/nonprofits/confirmation");
+        }
       })
       .catch((error) => {
-        console.log("There was a problem submitting the form:", error);
+        router.push("http://localhost:3000/nonprofits/errorOnSubmit");
```

Comment:
Same as above, don't want to hardcode localhost here - this will break on production